libr/crypto/crypto.c

```diff
@@ -194,7 +194,7 @@ static inline void print_plugin_verbose(RCryptoPlugin *cp, PrintfCallback cb_pri
 	cb_printf ("%c %12s %5s %s %s\n", type, cp->meta.name, license, desc, author);
 }
 
-R_API void r_crypto_list(RCrypto *cry, R_NULLABLE PrintfCallback cb_printf, int mode) {
+R_API void r_crypto_list(RCrypto *cry, R_NULLABLE PrintfCallback cb_printf, int mode, RCryptoType type) {
```

Comment:
to keep abi stable you can combine mode and type by shifting the value and unshifting it here

Reply:
Ok I implemented that, it does not break the ABI anymore and I would change it for 5.6.9.